setup.py

```diff
@@ -8,11 +8,23 @@
     with open(requirementPath) as f:
         install_requires = f.read().splitlines()
 
+dev_requirements = [
+    'autopep8',
+    'flake8'
```

Comment:
Is there a difference?

Reply:
autopep8 lets you do automatic fixing of lint errors e.g. `autopep8 --in-place --aggressive --aggressive --recursive .` whereas flake8 only tells you what errors you have